catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/metacard/ListApplication.java

```diff
@@ -123,9 +124,16 @@ public void init() {
             IOUtils.copy(attachmentInfo.getStream(), temporaryFileBackedOutputStream);
 
             for (Splitter splitter : lookupSplitters(attachmentInfo.getContentType())) {
-              if (attemptToSplitAndStore(
-                  response, listType, attachmentInfo, temporaryFileBackedOutputStream, splitter))
-                break;
+              try {
+                if (attemptToSplitAndStore(
+                    response, listType, attachmentInfo, temporaryFileBackedOutputStream, splitter))
+                  break;
+              } catch (StopSplitterExecutionException e) {
+                LOGGER.debug("Mime type error.", e);
```

Comment:
The exception reason could be something else. I would just log that the split failed.